### PR TITLE
131 Split up the components JSON

### LINKED ISSUE #131: Split up the components JSON

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #131 

Test URLs:
- Before: https://main--{repo}--{owner}.hlx.live/
- After: https://<branch>--{repo}--{owner}.hlx.live/

### CODE REVIEW

File: blocks/accordion/_accordion.json
Comment (mhaack) [inline blocks/accordion/_accordion.json:1]: Is this empty intentionally?

File: blocks/alert-banner/_alert-banner.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      "fields": [
+        {
+          "component": "text",
+          "valueType": "string",
+          "name": "alert-banner_title",
+          "value": "",
+          "label": "Title"
+        },
+        {
+          "component": "select",
```
Comment (mhaack) [inline blocks/alert-banner/_alert-banner.json:32]: A `select` should have some options to select.

File: blocks/hero-video/_hero-video.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        },
+        {
+          "component": "richtext",
+          "name": "hero-video_description",
+          "value": "",
+          "label": "Description",
+          "valueType": "string"
+        },
+        {
+          "component": "reference",
```
Comment (mhaack) [inline blocks/hero-video/_hero-video.json:73]: Using `image_` will render all 4 images into one table cell. This is maybe not what you want? Like what is user only selects image 1 & 3? You will get 2 images in one cell but you don't know if these are 1, 2, 3, or 4.

File: models/_common.json
Comment (mhaack) [inline models/_common.json:1]: You have bunch of block using image & imageAlt, you can make that re-useable here we well.

Comment (rgravitvl) [reply models/_common.json:1]: done

**Pull request merged**